### PR TITLE
[IMP] accounting: peppol

### PR DESCRIPTION
**Pull request by aliyatastemirova**
Recently the Peppol registration flow has been updated and new features were added. This commit:
- Updates the registration flow description in Peppol documentation
- Explains the decoupling between a sender and a receiver

task-3964853

### CODE REVIEW

**Comment by toaa-odoo** [comment]
Hi @aliyatastemirova, thanks for your great work! I added some details and changed a few things to comply with the guidelines. Passing it on to the final reviewer :)

**Review by toaa-odoo — APPROVED** [review]

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 1]
```diff
@@ -126,79 +126,100 @@ Registration
...
 Go to :menuselection:`Accounting --> Configuration --> Settings`. If you do not have the
 Peppol module installed, first tick the :guilabel:`Enable PEPPOL` checkbox and then **manually
-save**.
+save**. Click :guilabel:`Start sending via Peppol` to open the registration form.
 
-.. image:: electronic_invoicing/peppol-install.png
-   :alt: Peppol module installation
+.. note::
+  This registration form also pops up if you choose to **Send & Print** on an invoice via Peppol
+  without completing the registration process.
```
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:133]:
```suggestion
.. note::
   This registration form also pops up if you choose to :guilabel:`Send & Print` an invoice via Peppol
   without completing the registration process.
```

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 2]
```diff
@@ -222,6 +243,12 @@ It is possible to verify the Peppol participant status of several customers at o
 To do so, go to :menuselection:`Accounting --> Customers --> Customers` and switch to the list view.
 Select the customers you want to verify and then click :menuselection:`Actions --> Verify Peppol`.
 
+If the participant is registered on the Peppol network but cannot receive the format you selected
+for them, the :guilabel:`Peppol endpoint validity` label changes to **Cannot receive this format**.
```
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:247]:
```suggestion
for them, the :guilabel:`Peppol endpoint validity` label changes to :guilabel:`Cannot receive this format`.
```

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 3]
```diff
@@ -126,79 +126,100 @@ Registration
...
+   - For **advanced users only**, it is possible to run tests on Peppol's test network. The server
+     allows to register on Peppol, send, and receive test invoices to and from other
+     participants. To do so, enable the :doc:`../../../general/developer_mode`, open the
+     **Settings** app, go to :menuselection:`Technical --> System Parameters`, and search for
+     `account_peppol.edi.mode`. Open the parameter and change the :guilabel:`Value` to `test`. Go
+     back to the Peppol setup menu in the **Settings** app. The option :guilabel:`Test` is now
+     available.
 
    .. image:: electronic_invoicing/peppol-system-parameter.png
       :alt: Peppol test mode parameter
```
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:177]: I would remove this screenshot (and the corresponding image file). We're trying to limit screenshots as much as possible and I don't think this one adds much value to the content here.

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing/peppol-sender.png
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing/peppol-sender.png:1]: From what I can see, this image is not used so I would delete the file.

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 4]
```diff
@@ -126,79 +126,100 @@ Registration
...
-- :guilabel:`Peppol EAS`. This is the Peppol Electronic Address Scheme and usually depends on your
-  company's country. Odoo often prefills this with the most commonly used EAS code in your country.
-  For example, the preferred EAS code for most companies in Belgium is 0208.
-- :guilabel:`Peppol Endpoint`. This is usually a Company Registry number or a VAT number.
-- :guilabel:`Phone Number`, including the country code (e.g., `+32` in Belgium)
-- :guilabel:`Primary contact email`
+You can register either as sender or receiver. A sender can only send invoices and credit notes on
+Odoo via Peppol, without ever registering as a Peppol participant on Odoo SMP. If you have an
+existing Peppol registration elsewhere that you want to keep, but want to send invoices from your
+Odoo database and receive other documents in another software, register as **sender**.
```
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:141]:
```suggestion
existing Peppol registration elsewhere that you want to keep but want to send invoices from your
Odoo database and receive other documents in another software, register as a **sender**.
```

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 5]
```diff
@@ -126,79 +126,100 @@ Registration
...
 
-.. seealso::
-   - `Peppol EAS - European Commision <https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/Code+lists/>`_
-   - `Peppol Endpoint - OpenPeppol eDEC Code Lists <https://docs.peppol.eu/edelivery/codelists/>`_
-     (open the "Participant Identifier Schemes" as HTML page)
+.. tip::
+   - You can always register as a sender first and register to receive documents later.
+   - You can specify when registering if you would also like to receive documents.
```
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:145]:
```suggestion
   - When registering, you can specify if you would also like to receive documents.
```

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 6]
```diff
@@ -126,79 +126,100 @@ Registration
...
 
-.. note::
-   When testing Peppol, the system parameter `account_peppol.edi.mode` can be changed to `test`.
-   Then, a radio button appears with the option to register on the test server.
+.. tip::
+   - Selecting :guilabel:`Demo` simulates everything in Odoo. There is no sending, receiving, or
+     partner verification.
+   - For **advanced users only**, it is possible to run tests on Peppol's test network. The server
+     allows to register on Peppol, send, and receive test invoices to and from other
+     participants. To do so, enable the :doc:`../../../general/developer_mode`, open the
```
Comment (auva-odoo) [inline content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:170]:
```suggestion
     allows to register on Peppol and send/receive test invoices to/from other
     participants. To do so, enable the :doc:`../../../general/developer_mode`, open the
```
I tried to rephrase it to avoid the repetition of "and," but I couldn't find a better option. The original one wasn't quite right grammatically.

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hello @aliyatastemirova thank you very much for your work! I've added a few suggestions to reword a couple of sentences and correct a few minor oversights. There are also some images that need to be deleted as they're not being used (from what I can see anyways).

Let me know if you have any questions or need further clarification on any of the changes!

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 7]
```diff
@@ -126,79 +126,100 @@ Registration
...
+- :guilabel:`E-Address Scheme`: the Peppol Electronic Address Scheme and usually depends on your
+  company's country. Odoo often prefills this with the most commonly used EAS code in your country.
+  For example, the preferred EAS code for most companies in Belgium is `0208`.
+- :guilabel:`Endpoint`: this is usually a Company Registry number or a VAT number.
+- :guilabel:`Phone`: phone number including the country code (e.g., `+32` in Belgium).
+- :guilabel:`Email`: this the email Odoo can contact you to regarding your Peppol registration.
 
-   .. image:: electronic_invoicing/peppol-demo-mode.png
-      :alt: Peppol demo mode selection
+If you want to explore or demo Peppol, you can choose to register in :guilabel:`Demo` mode.
```
[the remark this answers is not in the document]
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:162]: this one is specifically about demoing it to someone :grimacing: "test" in this specific situation means testing on the test network

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 8]
```diff
@@ -126,79 +126,100 @@ Registration
...
-.. image:: electronic_invoicing/phone-registration.png
+.. image:: electronic_invoicing/peppol-phone-verification.png
    :alt: phone validation
 
-Once you enter the code and click :guilabel:`Confirm`, you can see that your registration is pending
-activation. From this point onwards, the default journal for receiving Vendor Bills can be set.
-
-.. image:: electronic_invoicing/peppol-registration-pending.png
-   :alt: pending application
+Once you enter the code and click :guilabel:`Register`, your Peppol participant status is updated.
```
[the remark this answers is not in the document]
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:191]: not sure about that, because you don't really click *in order to* update the status :thinking: the status is updated because things are happening behind the scenes, but registering is technically just about registering on our access point

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 9]
```diff
@@ -126,79 +126,100 @@ Registration
...
-.. image:: electronic_invoicing/peppol-registration-active.png
-   :alt: active application
+.. image:: electronic_invoicing/peppol-receiver.png
+   :alt: receiver application
 
 All invoices and vendor bills can now be sent directly using the Peppol network.
 
+.. important::
+   If you need to update the email that our team can use to contact you, you need to click
+   :guilabel:`Update contact details` for the updated data to be sent to us.
```
[the remark this answers is not in the document]
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:213]: I wanted to emphasize that nothing would happen unless they clicked that button because the data needs to be updated on our server. But if you think it's clear to users like that, I don't mind.

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing/peppol-services.png
[the remark this answers is not in the document]
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing/peppol-services.png:1]: added the image in the documentation :slightly_smiling_face:

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst (hunk 3 above)
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst:177]: I have just linked that part to a customer yesterday where they were confused about system parameters... so I think it could be useful?

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst [hunk 10]
```diff
@@ -126,79 +126,100 @@ Registration
...
+
+Configure Peppol Services
+-------------------------
+
+Once you are registered on Odoo SMP, a new button becomes visible: :menuselection:`Settings -->
+Configure Peppol Services`. Here, you can enable or disable document formats that other participants
```
[the remark this answers is not in the document]
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst]: yes, you're right :slightly_smiling_face: thanks
it only appears if you are in live mode, registered as a receiver :slightly_smiling_face:

File: content/applications/finance/accounting/customer_invoices/electronic_invoicing/peppol-services.png
[the remark this answers is not in the document]
Comment (aliyatastemirova) [reply content/applications/finance/accounting/customer_invoices/electronic_invoicing/peppol-services.png:1]: not sure why it got deleted...this is a new feature, one that is not easy to explain and one that is part of a feature that a lot of users were waiting for, but ok :thinking:

**Review by auva-odoo — APPROVED** [review]
Thank you @aliyatastemirova! I just force-pushed some really minor changes. 
I also:
- removed the image peppol-services because I don't really feel like it adds much and adds unnecessary maintenance (since the list of formats is very likely to change in the future). 
- cropped the existing image peppol-contact-verify to mirror the peppol-participant-format that's under it.

Hope this is ok! If it is, you can r+ whenever you're ready :slightly_smiling_face: Thanks again!
@robodoo delegate+

**Comment by aliyatastemirova** [comment]
@auva-odoo thanks, it's ok for me
@robodoo r+
